README.md

```diff
@@ -156,12 +156,12 @@ and favicon files into the markup.
 plugins: [
   new HtmlWebpackPlugin({
     title: 'Custom template',
-    template: 'my-index.ejs', // Load a custom template (ejs by default see the FAQ for details)
+    template: 'my-index.html', // Load a custom template (lodash by default see the FAQ for details)
```

Comment:
I think that would be better, we get so many issues just because they are using the `.html` ext and don't understand what's happening (and tbw most shouldn't care)